backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -829,4 +835,23 @@ private OrganizationRoles getOrgRolesFromDB(ApiUser apiUser) {
         _dbOrgRoleClaimsService.getOrganizationRoleClaims(apiUser);
     return _orgService.getOrganizationRoles(orgRoleClaims);
   }
+
+  private String getOrgExternalId(ApiUser apiUser) {
+    String orgExternalId;
+    if (_featureFlagsConfig.isOktaMigrationEnabled()) {
+      Optional<Organization> org = apiUser.getOrganizations().stream().findFirst();
```

Comment:
Do we have users with more than one organization?  I don't think we do, but do we know that for certain / are there guardrails in place to ensure that can't happen?